azext_iot/operations/hub.py

```diff
@@ -1618,6 +1618,44 @@ def iot_hub_configuration_delete(
         handle_service_exception(e)
 
 
+def iot_hub_configuration_test_queries(
+    cmd,
+    target_condition=None,
+    custom_metric_queries=None,
+    hub_name=None,
+    resource_group_name=None,
+    login=None,
+    auth_type_dataplane=None
+):
+    discovery = IotHubDiscovery(cmd)
+    target = discovery.get_target(
+        resource_name=hub_name,
+        resource_group_name=resource_group_name,
+        login=login,
+        auth_type=auth_type_dataplane,
+    )
+    resolver = SdkResolver(target=target)
+    service_sdk = resolver.get_sdk(SdkType.service_sdk)
+
+    try:
+        if custom_metric_queries:
+            custom_metric_queries = process_json_arg(custom_metric_queries, argument_name="custom_metric_queries")
+
+        result = service_sdk.configuration.test_queries(target_condition, custom_metric_queries, raw=True).response.json()
```

Comment:
What happens when target_condition is None?

Reply:
when target_condition is None, it will go ahead validate custom_metric_queries; if custom_metric_queries is also None, then the return will be {'Message': 'ErrorCode:ArgumentInvalid;BadRequest', 'ExceptionMessage': 'Tracking ID:cffd76d19a36464ca394469eda49b357-G:0-TimeStamp:01/04/2023 18:04:02'}